src/sdk/python/rtdip_sdk/pipelines/_pipeline_utils/iso.py

```diff
@@ -46,6 +46,25 @@
     ]
 )
 
+PJM_PRICING_SCHEMA = StructType(
+    [
+        StructField("StartTime", TimestampType(), True),
+        StructField("DatetimeBeginningEpt", TimestampType(), True),
```

Comment:
Can we remove DatetimeBeginningEpt column, as StartTime already has the same info just in UTC